release-information/release-notes/6_0_0/6_0_0-release-notes-core-changes.md

```diff
@@ -26,6 +27,10 @@ The following new features were added for the {{site.productname}} 6.0 release.
 
 The following enhancements were made for the {{site.productname}} 6.0 release.
 
+### `tabindex` improvement
+
+In iframe mode, TinyMCE now copies the [`tabindex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) attribute from the target element to the editor's iframe. This accessibility improvement allows integrators to rely on the browser behaviour for specifying tab order, instead of specially configuring TinyMCE. It also replaces the [Tabfocus plugin](#tabfocusplugin).
```

Comment:
```suggestion
In iframe (classic editor) mode, {{site.productname}} now copies the [`tabindex` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) from the target element to the editor's iframe. This accessibility improvement allows integrators to rely on the browser behavior for specifying tab order, instead of specially configuring {{site.productname}}. It also replaces the [Tabfocus plugin](#tabfocusplugin).
```